### ISSUE TITLE
ACCESSIBILITY ISSUE: Change color for warnings and errors on Preview tab after slicing to black

### ISSUE THREAD

**Issue by seanbo** [bambulab/BambuStudio#8997]
**Is your feature request related to a problem? Please describe.**
This is an accessibility problem.  I am low vision and cannot reatd white text on yellow/orange background at all.  I have to have someone else come read it.

**Describe the solution you'd like**
The text should be high contrast to the background, preferably black.

**Describe alternatives you've considered**
There are none other than getting help.  Even the accesibility features on my computer donmt change it to helpfful colors.

**Additional context**
This is an accessibility issue and should be taken care of quickly as it greatly affects the usage by low-vision users and opossibly the color blind.
